build.gradle.kts

```
@@ -69,6 +69,50 @@ tasks.register("publishMiraiCoreArtifactsToMavenLocal") {
     )
 }
 
+tasks.register("verifyCompiledClasses") {
+    val projects = listOf("mirai-core-api", "mirai-core-utils", "mirai-console", "mirai-console-terminal")
+    group = "verification"
+    doFirst {
+        projects.asSequence().mapNotNull {
+            runCatching { project(it) }.getOrNull()
+        }.map { project ->
```

```suggestion
       projects.mapNotNull { findProject(it) }
            .map { project: Project ->
```

我注意到很多地方用`runCatching`来选择性地执行`project()`，其实有个`findProject` API可以用的。